docs/projects/ad485x_fmcz/index.rst

```diff
@@ -0,0 +1,277 @@
+
+  - external_clk = 100MHz
+
+
+.. image:: ad485x_fmcz_clock_path.svg
+   :width: 800
+   :align: center
+   :alt: AD4851_FMCZ clock path
+
+CPU/Memory interconnects addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The addresses are dependent on the architecture of the FPGA, having an offset
+added to the base address from HDL(see more at :ref:`architecture`).
+
+==================== ===============
+Instance             Zynq
+==================== ===============
+axi_ad485x           0x43c00000
+axi_pwm_gen          0x43d00000
+ad485x_dma           0x43e00000
+adc_clkgen           0x44000000
+==================== ===============
+
+SPI connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 25 25 25 25
+   :header-rows: 1
+
+   * - SPI type
+     - SPI manager instance
+     - SPI subordinate
+     - CS
+   * - PS
+   * - PS
+     - SPI 0
+     - AD485x
+     - 0
+
+Interrupts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below are the Programmable Logic interrupts used in this project.
+
+================ === ========== ===========
+Instance name    HDL Linux Zynq Actual Zynq
+================ === ========== ===========
+axi_ad9783_dma   10  54         86
+================ === ========== ===========
+
+These are the board-specific interrupts
+(found in :git-hdl:`here <projects/ad485x_fmc/common/ad485x_bd.tcl>`).
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+The sources available :git-hdl:`here </>`. To get the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository.
+
+Then go to the project location (**projects/ad485x_fmc/carrier**) and run the
+make command by typing in your command prompt (this example is for
+:xilinx:`zed`):
+
+**Linux/Cygwin/WSL**
+
+.. code-block::
+
+   user@analog:~$ cd hdl/projects/ad485x_fmc/zed
+   user@analog:~$ make
+
+A more comprehensive build guide can be found in the :ref:`build_hdl` user
+guide.
+
+Resources
+-------------------------------------------------------------------------------
+
+Systems related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Here you can find the quick start guides available for these evaluation boards:
+
+.. list-table::
+   :widths: 20 10
+   :header-rows: 1
+
+   * - Evaluation board
+     - Zynq-7000
+   * - AD485x_FMCZ
+     - :dokuwiki:`zed <resources/fpga/xilinx/fmc/ad485x>`
+
+Hardware related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Product datasheets: :adi:`AD4858`
+- :dokuwiki:`EVAL-AD4858 user guide <resources/eval/ad4858_fmcz>`
+- Product datasheets: :adi:`AD4857`
+- :dokuwiki:`EVAL-AD4857 user guide <resources/eval/ad4858_fmcz>`
```

Comment:
Are these user guide links necessary? They don't exist now and I don't think we'll create more documentation to be posted on wiki

Reply:
I removed those links.